WordPress/Classes/Utility/DisplayableImageHelper.m

```diff
@@ -109,6 +109,34 @@ + (NSString *)searchPostContentForImageToDisplay:(NSString *)content
     return imageSrc;
 }
 
++ (NSSet *)searchPostContentForAttachmentIdsInGalleries:(NSString *)content
+{
+    NSMutableSet *resultSet = [NSMutableSet set];
+    // If there is no image tag in the content, just bail.
+    if (!content || [content rangeOfString:@"[gallery "].location == NSNotFound) {
+        return resultSet;
+    }
+
+    // Get all the things
+    static NSRegularExpression *regexGallery;
+    static dispatch_once_t onceTokenRegexGallery;
+    dispatch_once(&onceTokenRegexGallery, ^{
+        NSError *error;
+        NSString *galleryPattern = @"(\\[gallery [^]]*\\])";
```

Comment:
Maybe tweak the regex pattern so the capture group is on the IDs.  That would save a bit of work in `idsFromGallery`.  I'm not sure of the exact regex pattern off hand but I *think* it would be something like: 
```
@"\\[gallery[\s\S]+ids=\\"([0-9,]*)\\"[^]]*\\]"
```